### PR TITLE
feat(atom/skeleton): add skeleton component

### PR DESCRIPTION
**Pull request by andresz1**
**Is your component proposal related to a problem? Please describe.**
There are cases where spinners are not good enough to show a loading state properly, skeletons are used to display a placeholder preview of your content while avoiding layout shift and reducing frustration. 

**Describe the solution you'd like**
I'd like a component that I could use to create complex placeholders of a user interface.

**Screenshots**
<p align="center">

</p>

### CODE REVIEW

**Review by javiauso — APPROVED** [review]

File: components/atom/skeleton/src/index.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  return (
+    <>
+      {Array.from(Array(count).keys()).map(index => (
+        <span
+          key={index}
+          className={className}
+          style={{...style, width, height}}
+          {...others}
+        >
+          &zwnj;
```
Comment (davecarter) [inline components/atom/skeleton/src/index.js:42]: I'd rather use `$nbsp;` as a filler... Just wondering what's the difference?

**Review by davecarter — APPROVED** [review]
👏 👏 👏

**Review by Sabri1209 — APPROVED** [review]

File: components/atom/skeleton/src/index.scss [hunk 2]
```diff
@@ -0,0 +1,59 @@
+@import '~@s-ui/theme/lib/index';
+
+$bg-skeleton: $c-gray-light-3 !default;
+$bg-skeleton-highlight: $c-gray-light-4 !default;
+$bdrs-skeleton: 8px !default;
```
Comment (turolopezsanabria) [inline components/atom/skeleton/src/index.scss:5]: I suggest `$bdrs-base` here
you can fin it here: https://github.com/SUI-Components/sui-theme/blob/master/src/settings/_box-style.scss

File: components/atom/skeleton/src/index.scss [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  50% {
+    opacity: 0.5;
+  }
+  100% {
+    opacity: 1;
+  }
+}
+
+.react-AtomSkeleton {
+  background-color: $bg-skeleton;
```
Comment (turolopezsanabria) [inline components/atom/skeleton/src/index.scss:31]: use $bgc, as bg is the shorthand for background as general

**Review by turolopezsanabria — CHANGES_REQUESTED** [review]
super nice to see this PR! I've made some minor comments about our naming convention.
thnx

**Comment by midudev** [comment]
Let's continue here: https://github.com/SUI-Components/sui-components/pull/1219

**Pull request closed without merging**